packages/expo-task-manager/android/src/main/AndroidManifest.xml

```diff
@@ -8,12 +8,15 @@
             <intent-filter>
                 <action android:name="expo.modules.taskManager.TaskBroadcastReceiver.INTENT_ACTION" />
                 <action android:name="android.intent.action.BOOT_COMPLETED" />
+                <action android:name="android.intent.action.MY_PACKAGE_REPLACED" />
```

Comment:
❓

Reply:
Currently, while updating native app all alarms gets cancelled. If we want them to survive, we need to reschedule them after reinstall. This system broadcasts informs newly installed app of the fact it is reinstalled. Effectively, we have no Alarms, just like after reboot, so we need to reschedule them in exactly the same way as after reboot.